### PR TITLE
add a skeleton for the docs site

### PR DESCRIPTION
**Pull request by devoncarew**
- add a skeleton for the docs site

screen shot 2019-02-09 at 11 07 00 am

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Start an application to debug
+
+Next, start an app to connect to. This can be either a Flutter application or a Dart
+command-line application. The example below uses a Flutter app:
+
+- `cd path/to/flutter/app`
+- `flutter run --observatory-port=9200`
+
+You'll need to have a device connected - or a simulator open - for `flutter run` to work.
+Once the app starts you'll be able to connect to it from DevTools.
```
Comment (kenzieschmoll) [inline docs/cli.md]: nit: comma after "starts"

File: docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+browser, or run:
+
+```
+open http://localhost:9100/?port=9200
+```
+
+from the command line.
+
+In the above url, the first port is for the local server serving the DevTools web UI. The
+second port is to tell DevTools itself which local app to connect to in order to debug and
```
Comment (kenzieschmoll) [inline docs/cli.md]: nit: r/is to tell DevTools itself/tells DevTools

File: docs/cli.md (hunk 1 above)
Comment (devoncarew) [reply docs/cli.md]: done

File: docs/cli.md (hunk 2 above)
Comment (devoncarew) [reply docs/cli.md]: done

**Pull request merged**